### ISSUE TITLE
[Bug] `kopia repository create --help-full` is listed but doesn't work

### ISSUE THREAD

**Issue by sssheridan** [kopia/kopia#2290]
At [this page](https://kopia.io/docs/reference/command-line/common/repository-create-filesystem/) and output by `kopia repository create`, there's a flag listed:
````
  --[no-]help-full           Show help for all commands, including hidden
````

However, it produces:
````
$  kopia repository create --help-full
kopia: error: must select a subcommand of 'repository create', try --help
````

**Comment by basldfalksjdf** [comment]
Nit: I think you mean the page https://kopia.io/docs/reference/command-line/flags/

Also, I think `--help-full` is intended to reveal all the commands available at https://kopia.io/docs/reference/command-line/advanced/

Can you modify this PR to be a bug report, so someone can fix this? Should be an easy fix.

**Comment by sssheridan** [comment]
> Nit: I think you mean the page https://kopia.io/docs/reference/command-line/flags/

My bad, my initial report was sloppy: meant to say the output of `kopia repository create --help`, and I mis-assumed that the webpage had the same content.

Let me try again, more carefully:

When I run `kopia <any-command> <any subcommand> --help`, it outputs first a generic list of flags which apply to the `kopia` command, and then a context-sensitive list of flags which apply to `kopia <command> <subcommand>`, with no break or signifier between them.

For example:
````
> kopia policy set --help
usage: kopia policy set [<flags>] [<target>...]

Set snapshot policy for a single directory, user@host or a global policy.

Flags:
      --[no-]help                Show context-sensitive help (also try --help-long and --help-man).
      --[no-]version             Show application version.
      --log-file=LOG-FILE        Override log file.
      --log-dir="/home/steve/.cache/kopia"
                                 Directory where log files should be written. ($KOPIA_LOG_DIR)
      --log-level=info           Console log level
      --file-log-level=debug     File log level
      --[no-]help-full           Show help for all commands, including hidden
      --config-file="repository.config"
                                 Specify the config file to use ($KOPIA_CONFIG_PATH)
  -p, --password=PASSWORD        Repository password. ($KOPIA_PASSWORD)
      --[no-]persist-credentials
                                 Persist credentials ($KOPIA_PERSIST_CREDENTIALS_ON_CONNECT)
      --[no-]use-keyring         Use Gnome Keyring for storing repository password. ($KOPIA_USE_KEYRING)
      --[no-]global              Select the global policy.
      --inherit=INHERIT ...      Enable or disable inheriting policies from the parent
      --before-folder-action=COMMAND
                                 Path to before-folder action command ('none' to remove)
      --   (...)
````

Everything down to `--[no-]use-keyring` is the standard, non-context-sensitive list; everything below that is specific to `kopia policy set`.

Here's `snapshot estimate`:
````
> kopia snapshot estimate --help
usage: kopia snapshot estimate [<flags>] <source>

Estimate the snapshot size and upload time.

Flags:
      --[no-]help             Show context-sensitive help (also try --help-long and --help-man).
      --[no-]version          Show application version.
      --log-file=LOG-FILE     Override log file.
      --log-dir="/home/steve/.cache/kopia"
                              Directory where log files should be written. ($KOPIA_LOG_DIR)
      --log-level=info        Console log level
      --file-log-level=debug  File log level
      --[no-]help-full        Show help for all commands, including hidden
      --config-file="repository.config"
                              Specify the config file to use ($KOPIA_CONFIG_PATH)
  -p, --password=PASSWORD     Repository password. ($KOPIA_PASSWORD)
      --[no-]persist-credentials
                              Persist credentials ($KOPIA_PERSIST_CREDENTIALS_ON_CONNECT)
      --[no-]use-keyring      Use Gnome Keyring for storing repository password. ($KOPIA_USE_KEYRING)
      --[no-]show-files       Show files
  -q, --[no-]quiet            Do not display scanning progress
      --upload-speed=mbit/s   Upload speed to use for estimation
      --max-examples-per-bucket=10
                              Max examples per bucket

Args:
  <source>  File or directory to analyze.
````
Again, the flags specific to `kopia snapshot estimate` start after the ` --[no-]use-keyring` flag.

Am I right in thinking that this is unintended behavior? If so, please let me know where/how I should file this. (I see this current topic listed under "Issues," not "Pull requests" - is it really listed as a PR?)

**Comment by basldfalksjdf** [comment]
Sorry, I typoed that. This isnt a PR. I just meant put something like "[Bug]" in the title, so it is easy to spot for the devs
